test/NewsService.Mappers.UnitTests/ResponsesMappers/NewsResponseMapperTests.cs

```diff
@@ -21,120 +19,131 @@ namespace LT.DigitalOffice.NewsService.Mappers.UnitTests.ResponsesMappers
-        private Mock<IOperationResult<IGetUserDataResponse>> _responseMock;
+        private Mock<IRequestClient<IGetUserDataRequest>> _userRequestClientMock;
+        private Mock<IRequestClient<IGetDepartmentRequest>> _departmentRequestClientMock;
+        private Mock<Response<IOperationResult<IGetUserDataResponse>>> _userBrokerResponseMock;
+        private Mock<Response<IOperationResult<IGetDepartmentResponse>>> _departmentBrokerResponseMock;
+        private Mock<IOperationResult<IGetUserDataResponse>> _userResponseMock;
+        private Mock<IOperationResult<IGetDepartmentResponse>> _departmentResponseMock;
 
         private const string firstName = "Ivan";
         private const string lastName = "Ivanov";
         private const string middleName = "Ivanovich";
+        private const string departmentName = "Name";
 
         private INewsResponseMapper _mapper;
-        private NewsResponse _response;
+        private NewsResponse _newsResponse;
         private DbNews _dbNews;
         private User _user;
+        private Department _department;
 
-        [OneTimeSetUp]
-        public void OneTimeSetUp()
+        [SetUp]
+        public void SetUp()
         {
-            _user = new User
-            {
-                Id = Guid.NewGuid(),
-                FIO = "Ivanov Ivan Ivanovich"
-            };
+            _user = new User { Id = Guid.NewGuid(), FullName = "Ivanov Ivan Ivanovich" };
 
-            _response = new NewsResponse
-            {
-                Id = Guid.NewGuid(),
-                Content = "Content111",
-                Subject = "Subject111",
-                Author = _user,
-                Sender = _user,
-                CreatedAt = DateTime.UtcNow
-            };
+            _department = new Department { Id = Guid.NewGuid(), Name = departmentName };
 
             _dbNews = new DbNews
             {
-                Id = _response.Id,
+                Id = Guid.NewGuid(),
                 Content = "Content",
                 Subject = "Subject",
                 Pseudonym = "Pseudonym",
                 AuthorId = _user.Id,
                 SenderId = _user.Id,
                 CreatedAt = DateTime.UtcNow,
+                DepartmentId = _department.Id,
                 IsActive = true
             };
-        }
 
-        [SetUp]
-        public void SetUp()
-        {
-            _responseMock = new Mock<IOperationResult<IGetUserDataResponse>>();
+            _newsResponse = new NewsResponse
+            {
+                Id = _dbNews.Id,
+                Content = _dbNews.Content,
+                Subject = _dbNews.Subject,
+                Author = _user,
+                Sender = _user,
+                CreatedAt = _dbNews.CreatedAt,
+                Department = _department,
+                IsActive = _dbNews.IsActive
+            };
+
+            _userResponseMock = new Mock<IOperationResult<IGetUserDataResponse>>();
 
-            _responseMock
+            _userResponseMock
```

Comment:
It is too big setUp, mb would be better add regions, like UserMocks, DepartmentMocks, ModelsMocks, or smth like this

Reply:
done